proto3-wire.cabal

```
@@ -1,80 +1,108 @@
-name:                proto3-wire
-version:             1.4.1
-synopsis:            A low-level implementation of the Protocol Buffers (version 3) wire format
-license:             Apache-2.0
-license-file:        LICENSE
-author:              Awake Networks
-maintainer:          opensource@awakenetworks.com
-copyright:           2016 Awake Networks
-category:            Codec
-build-type:          Simple
-cabal-version:       >=1.10
-extra-source-files:  CHANGELOG.md
+cabal-version: >=1.10
+
+name:         proto3-wire
+version:      1.4.1
+synopsis:     A low-level implementation of the Protocol Buffers (version 3) wire format
+license:      Apache-2.0
+license-file: LICENSE
+author:       Awake Networks
+maintainer:   opensource@awakenetworks.com
+copyright:    2016 Awake Networks
```

I'd recommend updating this as well.

```suggestion
author:       Arista Networks
maintainer:   awake-opensource@awakenetworks.com
copyright:    2023 Arista Networks
```